b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/idpSection/otherComponents/idpCreateModal/googleIdentityProvider.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Correct the year. Check other places too.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/c9fb90d7d7ab19441f2e0d135cf2a6f218b7c78b